README.md

```diff
@@ -1,2 +1,29 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+#### (a)
+The algorithm performs a search of the list to find if there are any duplicate items
+
+#### (b)
+The worst possible running time is O(n^2) because the program might have to run to full length linear searches to find a duplicate ( the duplicate items are the last 2 items)
```

Comment:
✔️ -- well spotted that for each element of the list you have to run a linear search